src/chrome/content/rules/MicroAdInc.com.xml

```diff
@@ -1,5 +1,6 @@
 <!-- 
 	MicroAd, Inc related rulesets:
+		- MicroAd.co.jp.xml
 		- MicroAd.jp.xml
 		- MicroAd.net.xml
 		- MicroAdInc.com.xml
```

Comment:
Can you remove the reference to MicroAdInc.com.xml here, because this would be a reference to itself.